test/function/target/file.cpp

```diff
@@ -179,4 +179,48 @@ namespace {
     EXPECT_NULL( target );
     EXPECT_ERROR_ID_EQ( STUMPLESS_ARGUMENT_EMPTY );
   }
+
+  TEST( FileTargetOpenTest, OptionOdelay ) {
+    const char *filename = "option-odelay.log";
+    struct stumpless_target *target;
+    struct stumpless_entry *entry;
+    struct stumpless_element *element;
+    struct stumpless_param *param;
+    size_t line_count = 3;
+    size_t i;
+
+    stumpless_set_default_options( STUMPLESS_OPTION_ODELAY );
+
+    target = stumpless_open_file_target( filename );
+    EXPECT_NO_ERROR;
+    EXPECT_NOT_NULL( target );
+    EXPECT_NOT_NULL( target->id );
+
+    entry = stumpless_new_entry( STUMPLESS_FACILITY_USER,
```

Comment:
you can change this to the fixture `create_entry` function that is used in other places in this pull request